Baekjoon/2231_1.py

```diff
@@ -2,25 +2,38 @@
 문제 링크:https://www.acmicpc.net/problem/2231
 카테고리:브루트포스
 문제해설:분해합의 생성자 구하기
+
+M의 각자리수 + M
+
+*523%10=3 append->(523-3)//10=52
+->*52%10=2 append-> (52-2)//10=5=b
+->if b<10->break
+
+216입력->198출력
 '''
-n=int(input())
 
-def func(m):
+N=int(input())
+
+def M_sum(M):
```

Comment:
```suggestion
def m_sum(M):
```

함수의 시작은 소문자가 컨벤션일 거에요